Examples/Modules/laser_injection_from_file/analysis3d.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 Andrew Myers, Axel Huebl, Luca Fedeli
+# Remi Lehe
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+
+
+# This file is part of the WarpX automated test suite. It is used to test the
+# injection of a laser pulse from an external binary file for DIM=3.
+#
+# - Generate an input binary file with a gaussian laser pulse.
+# - Run the WarpX simulation for time T, when the pulse is fully injected
+# - Compute the theory for laser envelope at time T
+# - Compare theory and simulation, for both envelope and central frequency
+
+import yt ; yt.funcs.mylog.setLevel(50)
+import numpy as np
+import matplotlib
+matplotlib.use('Agg')
+from scipy.signal import hilbert
+import glob
+import os
+
+#Maximum acceptable error for this test
+relative_error_threshold_envelope = 0.12
+relative_error_threshold_frequency = 0.05
+
+#Physical parameters
+um = 1.e-6
+fs = 1.e-15
+c = 299792458
+
+#Parameters of the gaussian beam
+wavelength = 1.*um
+w0 = 2.0*um
+tt = 10.*fs
```

Comment:
Using a shorter pulse will allow for a smaller box (possible a factor of 2)
```suggestion
tt = 5.*fs
```

Reply:
This will require other changes below and in the input file, let me know what you think.